crates/core/component/dao/src/component/state_key.rs

```diff
@@ -1,10 +1,14 @@
 use penumbra_asset::asset;
 
+pub fn dao_params() -> &'static str {
+    "dao_params"
```

Comment:
nit: we probably want this to be `dao/params` so that it's under the same prefix as the other keys in the component?